huaweicloud/services/acceptance/css/resource_huaweicloud_css_es_loadbalancer_config_test.go

```diff
@@ -0,0 +1,223 @@
+package css
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/css/v1/cluster"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getCssEsLoadbalancerConfigFunc(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := conf.NewServiceClient("css", acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating CSS v1 client: %s", err)
+	}
+
+	getEsListenerHttpUrl := "v1.0/{project_id}/clusters/{cluster_id}/es-listeners"
+
+	getEsListenerPath := client.Endpoint + getEsListenerHttpUrl
+	getEsListenerPath = strings.ReplaceAll(getEsListenerPath, "{project_id}", client.ProjectID)
+	getEsListenerPath = strings.ReplaceAll(getEsListenerPath, "{cluster_id}", state.Primary.ID)
+
+	getEsListenerOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			200,
+		},
+		MoreHeaders: map[string]string{"Content-Type": "application/json"},
+	}
+
+	getEsListenerResp, err := client.Request("GET", getEsListenerPath, &getEsListenerOpt)
+	if err != nil {
+		return nil, err
+	}
+
+	getEsListenerRespBody, err := utils.FlattenResponse(getEsListenerResp)
+	if err != nil {
+		return nil, err
+	}
+
+	return getEsListenerRespBody, nil
+}
+
+func TestAccCssEsLoadbalancerConfig_basic(t *testing.T) {
+	rName := acceptance.RandomAccResourceName()
+	resourceName := "huaweicloud_css_es_loadbalancer_config.test"
+
+	var obj cluster.ClusterDetailResponse
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&obj,
+		getCssEsLoadbalancerConfigFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCssEsLoadbalancerConfig_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(resourceName, "loadbalancer_id",
+						"huaweicloud_elb_loadbalancer.test", "id"),
+					resource.TestCheckResourceAttr(resourceName, "agency", "css_elb_agency"),
```

Comment:
how about add this agency to accentence as a variable, if it is deleted, then an error will be occured